docs/specification/draft/server/profiles.mdx

```diff
@@ -0,0 +1,137 @@
+
+## Profile Specification
+Each Profile has a specification document that describes the profile in detail. The profile specification is a document written using best practices prescribed in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) that unambiguously specifies the characteristics of a profile.
+
+### Profile URL
+
+A profile specification is available at the URL that is used to uniquely identify the profile. For example:
+
+```
+https://mycompany.com/mycustomprofile-1.0
+```
+
+OR
+
+```
+https://modelcontextprotocol.io/specification/2025-06-18/server/standard-profiles/minimum-authorization-profile
+```
+
+### Standard and Custom Profiles
+- Standard profiles are governed by the Model Context Protocol and stored in the "Standard Profiles" folder in the Server section of this specification. Standard profiles always have the profile URL prefix:
+  ```
+  https://modelcontextprotocol.io/specification/<version>/server/standard-profiles/
+  ```
+- Any profiles available outside of the standard profiles location are called Custom Profiles.
+- MCP Servers may simultaneously support a mix of standard and custom profiles.
+
+### Profile Modification and Versioning
+Once made available, Profiles MUST NOT be changed. In order to make changes to a Profile, the author MUST create a new version of the Profile, with its own Profile URL.
+
+## Publishing Profiles
+An MCP Server makes it possible for Clients to discover the profiles it supports as follows:
+
+### ProfileSpec Object
+
+A ProfileSpec object has the following fields:
+
+- The location of the profile specification
+- The minimum MCP version required to use that profile.
+
+```TypeScript
+interface ProfileSpec {
+  "profileURL": string; 
+  // The unique URL identifying the profile,
+  // which is also location where the profile is specified.
+
+  "minMcpVersion": string;
+  // this is the date in YYYY-MM-DD format that identifies the MCP version.
+}
+```
+
+### Profiles Declaration
+The profiles supported by an MCP Server are available through the Profiles Declaration JSON format document that is an array of `ProfileSpec` objects:
+
+```TypeScript
+[ ProfileSpec, ProfileSpec, ...]
+```
+
+#### Default Profile
+A default profile is one that the server uses if the client does not request a specific profile. The first `ProfileSpec` in the Profiles Declaratoin document is the Server's default profile.
+
+### Well-known URL
+The Profiles Declaration is made available by an MCP Server at its "well-known" location, as described in [RFC8615](https://datatracker.ietf.org/doc/html/rfc8615). The file name for the MCP Profiles Declaration document is: `mcp-profiles`. Thus the well-known URL for the MCP Profiles Declaration is obtained by inserting the path:
```

Comment:
This introduces a confusing structure - you've defined that the URL of a profile is the one earlier in the doc, but now we're stating that the profile is actually hosted by the server?